query/stdlib/testing/testing.go

```diff
@@ -62,8 +62,8 @@ var FluxEndToEndSkipList = map[string]map[string]string{
 		"keys":               "group key mismatch",
 
 		// failed to read meta data errors: the CSV encoding is incomplete probably due to data schema errors.  needs more detailed investigation to find root cause of error
-		"filter_by_regex":             "failed to read metadata",
-		"filter_by_tags":              "failed to read metadata",
+		// "filter_by_regex":             "failed to read metadata",
```

Comment:
Any reason not to just delete these lines?

Reply:
These probably should have been deleted. I will come back to this in the future, but I don't want to make additional modifications beyond updating the version in this PR.